### PR TITLE
Create the 'contact me' section

### PR DESCRIPTION
**Pull request by emmiiorji**
Following the [design template2](https://www.figma.com/file/l7SqJ3ZfkAKih9sFxvWSR4/Microverse-Student-Project-1?node-id=48%3A27) as closely as possible, I have converted the contact me section of the page to HTML and CSS.

Find below screenshots of my conversion:

image

image

Observations and comments are welcome.

### CODE REVIEW

File: index.html [hunk 1]
```diff
@@ -194,5 +194,31 @@ <h1>About me</h1>
...
+  <footer class="footer">
+    <p>I'm always interested in hearing about new projects, 
+      so if you'd like to chat please get in touch.
+    </p>
+
+    <form action="https://formspree.io/f/xjvleokg" method="POST">
+      <input type="text" placeholder="Full Name" maxlength="30" required>
+      <input type="email" placeholder="Email Address" required>
+      <textarea name="textarea" placeholder="Enter text here" maxlength="500" required></textarea>
+      <button type="submit">Get in touch</button>
```
Comment (MeqdamAlqudah) [inline index.html:208]: - [ ] Please stick with Figma design the textarea shouldn't have a border-bottom, I know this is might sound silly but in the feature, you will work in a company and you must 100% stick with the designs they give to you 
your design : 
image
Figma design : 
image

File: index.html [hunk 2]
```diff
@@ -194,5 +194,31 @@ <h1>About me</h1>
...
+
+      <hr id="footer-divide">
+
+      <ul class="socials">
+        <li><a href="https://github.com/emmiiorji"><i class="fa-brands fa-github"></i></a></li>
+        <li><a href="https://www.linkedin.com/in/emmanuel-orji-2a8317121"><i class="fa-brands fa-linkedin-in"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-angellist"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-twitter"></i></a></li>
+        <li><a href="#"><i class="fa-brands fa-medium"></i></a></li>
+      </ul>
```
Comment (MeqdamAlqudah) [inline index.html]: - [ ] Please use the same icons from Figma design you can simply click on the icon and download it then 
Figma desing : 
image

your desing : 
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @emmiiorji , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
-  Readme file✅
- no Linter errors ✅
- you got it from the first time ✅

## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by Wes-Isaac — APPROVED** [review]
Hi @emmiiorji, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**